SethBurns/Overlook
# Feature/render new bookings

## What is the feature?
Remove booked room from searched rooms. Book the room to the API, fetch from API to render newly booked rooms. Add message declaring room was booked successfully.
Closes #6 
## How you implemented the solution?
Wrote functionality for rendering the successful message. Refactored handleBookingClick to pull from API to render newly booked rooms.

## Does it impact any other area of the project?
No

## Any changes in the UI (Screenshots)?
![Screen Shot 2023-06-12 at 11 19 54 AM](https://github.com/SethBurns/Overlook/assets/123792434/835e4d3e-3bcd-4e2e-9e17-257f2f83c7d1)

## How to test the feature (A test scenario or any new setup)?
Search for room and click book now to render successful message and see new room appear in Booked Rooms ordered by most recent to oldest.
